packages/quantic/force-app/main/default/lwc/quanticSummary/quanticSummary.js

```diff
@@ -76,6 +79,16 @@ export default class QuanticSummary extends LightningElement {
 
   updateState() {
     this.state = this.querySummary.state;
+    if(this.state?.hasResults) {
+      this.updateAriaMessage();
+    }
+  }
+
+  updateAriaMessage() {
+    const docElement = document.createElement('div');
```

Comment:
why do we need this manipulation? is it because` this.summaryLabel` contains HTML tags?

Reply:
Yea the function I use to get what is going to be displayed inside the quanticSummary inserts <b> tags around the query text and things like that, since I didn't want to re-code another function to get the same results, I decided to just sanitize the html into text instead.

And yes the screen reader was actually reading the 🔊`smaller than b greater than query text smaller than forward slash b greater than`